src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -49,6 +49,9 @@ using namespace rapidjson;
 *
 * In the case of /v2/entities/<eid>, the entityId of 'Entity* eP' is set in
 * the service routine postEntity.
+*
+* Also, if the URI param 'options' includes the value 'keyValues', then the
+* parse changes to accept 'unknown fields' as attributes inside entities.
```

Comment:
"Also, if the URI param 'options' includes the value 'keyValues', then the parse changes to accept 'unknown fields' as attributes inside entities".

Not sure if the comment is fully correct... I mean, in no-keyValues mode, the attributes are also unkown fields (the difference is that the value of such unknown fields follows a given pattern). Maybe I don't understanding correctly...